data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/S3ScanBucketOptions.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+/**
+ * Class consists the bucket properties.
+ */
+public class S3ScanBucketOptions {
```

Comment:
Why do we need this class with single property ? Cant we use `S3ScanBucketOption` instead ?

Reply:
Thanks for your review Dinu John, As per the pipeline design, we have to define bucket list under the buckets property.

pipeline:
scan:
   start_time: 2023-04-15T18:00:00
   range: 3m
   **buckets**:
     - bucket:
          name: s3-source-test-output1
          ......
      - bucket:
          name: s3-source-test-output2
          ........
      - bucket:
          name: s3-source-test-output3
          .......